esbuild.server.js

```diff
@@ -0,0 +1,36 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const dotenv = require('dotenv');
+
+// Pulls in .env and populates the define object.
+// This is used to populate environment variables at built time.
+dotenv.config();
+const define = {};
+for (const k in process.env) {
+  // eslint-disable-next-line functional/immutable-data
+  define[`process.env.${k}`] = JSON.stringify(process.env[k]);
+}
+
+const watchMode = process.env['WATCH'];
```

Comment:
If `watchMode` is an environment variable it should be in upper case, no? Not sure where this convention comes from - it might just be in my head! - but it feels like a thing

Reply:
You could also just inline `process.env['WATCH']`. If you need to assign it to a variable to describe what it does that can be useful but maybe we could look at renaming the variable?

Reply:
I think inlining it is a good idea, it's quite self explanatory as-is